ocaml/libs/xapi-stdext/lib/xapi-stdext-threads/scheduler_test.ml

```diff
@@ -16,25 +16,41 @@ module Scheduler = Xapi_stdext_threads_scheduler.Scheduler
 
 let started = Atomic.make false
 
+let calibrated_ratio =
+  let expected = Mtime.Span.(100 * ms |> to_uint64_ns) |> Int64.to_float in
```

Comment:
Here and below, is there any reason not to use `Span.to_float_ns` instead of `to_uint64_ns |> Int64.to_float`?

Reply:
hmm, nope. Will change